system/system_error_monitor/config/diagnostic_aggregator/vehicle.param.yaml

```diff
@@ -14,3 +14,26 @@
               path: topic_status
               contains: [": vehicle_topic_status"]
               timeout: 1.0
+        # added
+        calibration:
+          type: diagnostic_aggregator/AnalyzerGroup
+          path: calibration
+          analyzers:
+            accel_brake_map_calibrator:
+              type: diagnostic_aggregator/AnalyzerGroup
```

Comment:
If you add this here, please generalize the name more.

Reply:
I will remove this change because this is unnecessary